### DISCUSSION TITLE
unique visitors per month/year

### DISCUSSION THREAD

**Discussion by jlmenut** [allinurl/goaccess#2669] [Q&A] [answered]
Hello,

As far as I understand, an unique visitor is a combination of IP+day of visit+user agent. So the same person visiting several successive days will be counted as as many unique visitor. Is there a way, directly or by processing a daily output of goaccess to increase the period considered ?  to one month for example ? And is it possible to completely drop the user agent  ? 
Thanks !

**Comment by allinurl** [comment, marked as answer]
Awesome idea! Unfortunately, it's not an option at the moment. But I believe we could incorporate this feature. Perhaps we could include a feature where users can specify the key, like `--unique-visitors-key=IP|%b/%Y|UA`, allowing for customization such as `IP|%d/%b/%Y`.  Thoughts?

**Reply by jlmenut** [reply]
Looks good !
